Owncloud iOs Client/DataBase/Queries/ManageDB.m

```diff
@@ -992,4 +993,31 @@ + (void) updateDBVersion16To17{
     
 }
 
+///-----------------------------------
+/// @name Update Database version with 17 version to 18
+///-----------------------------------
+
+/**
+ * Changes:
+ *
+ * Alter users table, adds new field to track user background Instant Upload preference
+ */
++ (void) updateDBVersion17To18 {
+    
+    BOOL defaultBackgroundInstantUploadValue = (BOOL)[ManageAppSettingsDB isInstantUpload]; //Users who are using Instant Upload before this migration also have background upload enabled, so we want to default to enabling this preference.
+    
+    FMDatabaseQueue *queue = Managers.sharedDatabase;
+    
+    [queue inTransaction:^(FMDatabase *db, BOOL *rollback) {
+        BOOL updateSuccessful;
+        
+        updateSuccessful = [db executeUpdate:[NSString stringWithFormat:@"ALTER TABLE users ADD background_instant_upload INTEGER DEFAULT %i", defaultBackgroundInstantUploadValue]];
```

Comment:
It is cleaner use:
updateSuccessful = [db executeUpdate:@"@"ALTER TABLE users ADD background_instant_upload INTEGER DEFAULT ?", [NSNumber numberWithBool: defaultBackgroundInstantUploadValue]];